doc/source/index.rst

```diff
@@ -1,4 +1,50 @@
 .. include:: ../../README.rst
+   :start-after: .. readme_common_begins
+   :end-before: .. readme_common_ends
+
+For more information on available actions and how to use them,
+see each card below:
+
+.. grid:: 1 1 2 2
+    :gutter: 2
+
+    .. grid-item-card:: :octicon:`codescan-checkmark` Style actions
+       :link: style-actions/index
+       :link-type: doc
+
+        Style actions verify code and documentation.
+
+    .. grid-item-card:: :octicon:`tools` Tests actions
+       :link: tests-actions/index
+       :link-type: doc
+
+        Tests actions allow to run the test suite.
+
+    .. grid-item-card:: :octicon:`book` Docs actions
+       :link: doc-actions/index
+       :link-type: doc
+
+        Documentation actions build and deploy the documentation.
+
+    .. grid-item-card:: :octicon:`key` Licence actions
+       :link: licenses-actions/index
+       :link-type: doc
+
+        Licence action sallows to verify that the project’s dependencies only contain valid licenses.
```

Comment:
Even though if we currently have one of these, let us use the plural as that is the name of the section.

```suggestion
        Licence actions sallows to verify that the project’s dependencies only contain valid licenses.
```